package.json

```diff
@@ -12,7 +12,7 @@
   "author": "Vizbee",
   "license": "ISC",
   "peerDependencies": {
-    "react-native": "^0.41.2"
+    "react-native": "^0.59.0"
```

Comment:
Why do we need to update this? Is the Pressable component available only from 0.59.0?
As per ChatGPT
```
The Pressable component, formerly known as TouchableWithoutFeedback, has indeed been available since React Native version 0.41.2. It was initially introduced as TouchableWithoutFeedback and then renamed to Pressable in React Native version 0.63.0.
```

Reply:
VizbeeCastButtonPressable/index.js uses functional component and hooks which require the upgrade

Reply:
@Sidharth-iOS What will be the impact on existing customer integrations (eg., NFL, Tegna etc) with this version change?
cc @shivareddy

Reply:
1. NFL is on "react-native": ">=0.60.5", 
2. NewsON I am sure they must be using > 0.59.0 as I have seen functional components in their code during screen share, but I can re-confirm with them
3. Tegna also I have seen functional components during the cast icon bug screen share but we need to check with them for confirmation. @naveenk1223 can you ask Tegna?
4. Are there any other customers?? - I don't think so.

Reply:
Also, the recent version of react-native is 0.74 considering this, 0.59 is a decent upgrade that we can make to get the advantages of functional components and I am sure, every other app must be using them.